web/src/AdapterPage.js

```diff
@@ -2,9 +2,7 @@ import React from "react";
 import * as Setting from "./Setting";
 import * as Backend from "./Backend";
 import {Button, Card, Col, Input, Row, Select, Tag} from "antd";
-import {Controlled as CodeMirror} from 'react-codemirror2'

```

Comment:
@aryamanpuri , the packages like `CodeMirror` are going to be required in the future as mentioned in the ideas list too and I guess that's why they have been imported in the codebase for now. I don't think removing them is a viable fix here!